Northstar.Custom/mod/scripts/weapons/mp_titanweapon_triplethreat.txt

```diff
@@ -414,6 +414,16 @@ WeaponData
     "clip_bodygroup_show_for_milestone_2"	"1"
 
 
+	// these fixes npc dealing damage
+	"npc_explosion_damage"							"350"
+	"npc_explosion_damage_heavy_armor"				"550"
+	// these fixes npc targeting enemies
+	// same as softball's status
+	"npc_min_engage_range"							"100"
+	"npc_max_engage_range"							"2000"
+	"npc_min_engage_range_heavy_armor"				"500"
+	"npc_max_engage_range_heavy_armor"				"3500"
```

Comment:
Maybe it would be better to use the Leadwall's stats here? I'm just a bit concerned that the softball's stats are skewed in some way towards being used by a human-sized AI

might not be a problem in actuality though, I havent tested

Reply:
that was a good idea! I'd leave min engage range to make npcs avoid hurting themselves, for max range I'm gonna use leadwall's so they won't waste ammo

Reply:
just noticed there's already a "npc_max_range" exists